mangaki/mangaki/management/commands/analytics.py

```diff
@@ -32,7 +32,11 @@ def handle(self, *args, **options):
             """if 'Baccano' in anime_map[anime].title:
                 print(anime_map[anime].title, (nb_likes, nb_dislikes))"""
             # if nb_dislikes <= 2:
-            score[(anime, (nb_likes, nb_dislikes))] = controversy(nb_likes, nb_dislikes)  # nb_likes
+            nb = {
+                'dislike': nb_dislikes,
+                'like': nb_likes
+            }
+            score[(anime, (nb_likes, nb_dislikes))] = controversy(nb)
```

Comment:
Not sure this is going to work, unless it was previously not working (which seems to be the case).
Actually, I think this command is broken since 68f7a8e35d2a37027d3d0999681a9ae14f373d8b i.e. over a year ago -- we can probably safely remove the whole file.